service/history/hsm/registry.go

```diff
@@ -145,17 +143,16 @@ func RegisterImmediateExecutor[T Task](
 	return nil
 }
 
-// RegisterExecutors registers an active and a standby [ImmediateExecutor] for the given task type.
+// RegisterTimerExecutor registers an active [TimerExecutor] for the given task type.
```

Comment:
```suggestion
// RegisterTimerExecutor registers a [TimerExecutor] for the given task type.
```

Reply:
You can fix it for `RegisterImmediateExecutor` too.

Reply:
I'll do it in another PR that will do more renaming

Reply:
I'd just fix at least this one while you're at it.